trimmer.rb

```diff
@@ -0,0 +1,8 @@
+require 'decorator'
+class TrimmerDecoratorr < Decorator
+  def correct_name
+    return unless @nameable.correct_name.size == 10
+
+    @nameable.correct_name.strip
+  end
+end
```

Comment:
- [ ] According to the project requirement if `@nameable.correct_name has a maximum of 10 characters. If it's longer it should trim the word.` but in your code, `it checks whether the size (length) of @nameable.correct_name is equal to 10. If it is not equal to 10, the method will immediately return nil (essentially doing nothing) because of the unless keyword, which means the subsequent code won't be executed if the condition is false.` Please make the changes as you can see the code below for correction
``` 
def correct_name
      @nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name
 end
```
what this code does it that if `@nameable.correct_name. length> 10 then it trim the length and returns it.